docs/Status_Reference.md

```diff
@@ -541,16 +541,36 @@ The following information is available in the `toolhead` object
   the printer had to be paused because the toolhead moved faster than
   moves could be read from the G-Code input.
 
+Additionally, if the printer uses `generic_cartesian` kinematic, the
+follwoing information is available:
+- `stepper_kinematics`: a map from a stepper name to the array of 3
+  floats - coefficients of the stepper kinematics for each of the axis
+  X, Y, and Z from the corresponding `kinematics` option, as indicated
+  by the specified carriages and their axes. Can be used to compute
+  stepper positions from the known toolhead position.
+- `toolhead_kinematics`: a map from a stepper name to the array of 3
+  floats - a contribution of a given stepper into the motion of each of
+  the axes X, Y and Z of the printer - which is effectively a
+  pseudoinverse of the `stepper_kinematics`. Can be used to compute
+  the toolhead position from the known stepper positions.
```

Comment:
This info seems very technical.  What's the use-case that users need this information for?

The printer.toolhead object has become a bit of a "dumping ground" for random bits of info and it's getting a little cluttered.  I'm not sure if we should add more there.  This is not a "blocker" though.

Reply:
OK, I've removed this for now (and naturally from the documentation)